package.json

```diff
@@ -37,15 +37,15 @@
     "@typescript-eslint/eslint-plugin": "^5.60.1",
     "@typescript-eslint/parser": "^5.60.1",
     "electron": "^24.4.1",
-    "electron-builder": "^23.6.0",
+    "electron-builder": "^25.0.5",
     "electron-vite": "^1.0.23",
-    "eslint": "^8.42.0",
+    "eslint": "^8.57.1",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-solid": "^0.12.1",
     "prettier": "^2.8.8",
     "solid-js": "^1.7.6",
-    "typescript": "^5.1.3",
+    "typescript": "5.0.4",
```

Comment:
Why does this PR downgrade Typescript?

Reply:
the eslint version installed (8.57.1) is not compatible with >=5.1.0
i will update that instead of downgrading typescript

Reply:
i updated every library
eslint just does not support the most recent typescript version
this warning will apear when running eslint but probably everything will work fine
![image](https://github.com/user-attachments/assets/eac946d0-5125-406a-bbe0-ff8fd4f01590)

Reply:
Typescript "^5.5.0" should work fine with the latest eslint.

See here for the current status of ESLint with Typescript 5.6.
https://github.com/typescript-eslint/typescript-eslint/issues/9653

Reply:
it still uses 5.6.2 because of the "^"

Reply:
iirc you can do `5.5.x` to lock the minor version

Reply:
My mistake, the caret allows minor upgrades, So we'd want to use `~5.5.0`, or `5.5.x` as @uzervlad mentioned.
^ allows minor and patch version bumps.
~ only allows patch version bumps.